File: Wox.UI.Flutter/wox/lib/entities/websocket_msg.dart
class WebsocketMsg {
  String? id;
  String? type;
  String? method;
  bool? success;
  dynamic data;

  WebsocketMsg({this.id, this.type, this.method, this.success, this.data});

  WebsocketMsg.fromJson(Map<String, dynamic> json) {
    id = json['Id'];
    type = json['Type'];
    method = json['Method'];
    success = json['Success'];
    data = json['Data'];
  }

  Map<String, dynamic> toJson() {
    return <String, dynamic>{
      'Id': id,
      'Type': type,
      'Method': method,
      'Success': success,
      'Data': data,
    };
  }
}

File: wox.ui.windows/Models/WebsocketMsg.cs
using System.Text.Json.Serialization;

namespace Wox.UI.Windows.Models;

public class WebsocketMsg
{
    [JsonPropertyName("RequestId")]
    public string RequestId { get; set; } = string.Empty;

    [JsonPropertyName("TraceId")]
    public string TraceId { get; set; } = string.Empty;

    [JsonPropertyName("SessionId")]
    public string SessionId { get; set; } = string.Empty;

    [JsonPropertyName("Method")]
    public string Method { get; set; } = string.Empty;

    [JsonPropertyName("Type")]
    public string Type { get; set; } = string.Empty;

    [JsonPropertyName("Data")]
    public object? Data { get; set; }

    [JsonPropertyName("Success")]
    public bool Success { get; set; }
}

public static class WebsocketMsgType
{
    public const string Request = "WebsocketMsgTypeRequest";
    public const string Response = "WebsocketMsgTypeResponse";
}
